src/xo-app/host/tab-patches.js

```diff
@@ -111,6 +111,26 @@ const INSTALLED_PATCH_COLUMNS_2 = [
   needsRestart: createDoesHostNeedRestart((_, props) => props.host)
 }))
 export default class HostPatches extends Component {
+  static contextTypes = {
+    router: React.PropTypes.object
+  }
+
+  _installPatchWarning = (patch, installPatch) => confirm(
+    {
```

Comment:
Either put this bracket on the previous line or the `.then` on the following line.
Same for `_installAllPatchesWarning`